examples/language-modeling/main.py

```diff
@@ -116,6 +116,10 @@
     parser.add_argument("--quant_lm_head", action='store_true',
                         help="quant_lm_head")
 
+    parser.add_argument("--layer_wise", action='store_true',
```

Comment:
is there anyway to consolidate it with low_cpu_mem_usage,  block_wise O1, layerwise O2?